app/javascript/dashboard/routes/dashboard/helpcenter/pages/articles/ArticleSettings.vue

```diff
@@ -48,6 +48,18 @@
             @click="onClickAssignAuthor"
           />
         </label>
+        <woot-input
```

Comment:
We can change the input type to number. Also the typography should match other fields like `meta title`